src/domains/DevTeam/devTeamData.ts

```diff
@@ -0,0 +1,179 @@
+import teamMember1 from 'assets/teamMembers/teamMember1.jpg';
```

Comment:
I forgot to mention this last time, but make sure to add a `.test` or `.mock` to this so it gets stripped from production.